e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/reconcile-fee-records.spec.js

```diff
@@ -16,10 +16,31 @@ context('PDC_RECONCILE users can reconcile fee records', () => {
   const REPORT_ID = 1;
   const FEE_RECORD_ID_ONE = '11';
   const FEE_RECORD_ID_TWO = '22';
+  const FACILITY_ID_ONE = '11111111';
+  const FACILITY_ID_TWO = '22222222';
+
+  const aTfmFacilityFacilitySnapshot = () => ({
+    coverStartDate: new Date().getTime(),
+    coverEndDate: new Date().getTime(),
+    interestPercentage: 5,
+    dayCountBasis: 365,
+  });
```

Comment:
Looks like this doesn't have all the required fields of the type it should really be returning which is a `Facility` I think, could we make it do so please?

Reply:
The reason I was hesitant to do that is that we don't have typescript enabled inside e2e-tests, so it would be really easy for a generic `aFacility` helper to fall out of sync with what we expect a facility to be

Reply:
Ah I had failed to realise we don't even have typescript for helper files in the e2e-tests, in that case I agree fine to leave this

Reply:
I have a few ideas on how we can reconcile that in a future fix for e2e tests, but it would also involve having to properly build `libs/common` which is why I'm holding off on opening any fix for it